alphadia/outputtransform.py

```diff
@@ -794,7 +798,9 @@ def build_lfq_tables(
         # as we want to retain decoys in the output we are only removing them for lfq
         qb = QuantBuilder(psm_df[psm_df["decoy"] == 0])
 
-        intensity_df, quality_df = qb.accumulate_frag_df_from_folders(folder_list)
+        dfs = qb.accumulate_frag_df_from_folders(folder_list)
+        intensity_df = dfs["intensity"].copy()
```

Comment:
Do we need to call copy here? this is quite expensive I think and might not be necessary.